### ISSUE TITLE
Adding volume manually - minimal option

### ISSUE THREAD

**Issue by plakitkelly** [keeleinstituut/tv-tolkevarav#684]
Click on Lisa maht in task tab
Choose Lisa maht käsitsi
Select Minimaalne

User sees Viga. Can't select Minimaalne option

![image](https://github.com/keeleinstituut/tv-tolkevarav/assets/130659408/c582fe68-043c-4cd5-8171-eb57b53deb31)

**Comment by MariusJulius** [comment]
@PlaksoBirgit is the logic like that: User adds minimum cost (can't edit any field) and when vendor is added it will display the price according to vendor price?

**Comment by PlaksoBirgit** [comment]
@MariusJulius it should be as with other options - initially when no vendor has accepted and "Miinimum tasu" is selected, system displays 0, but PM can add values (to estimate approximate cost of the sub-project) and save them.
After a vendor accepts the task then it should be retrieved from their pricelist and the initially added values will be over written, but if needed it can still be edited by PM. 
(in CAT tool volume modal editing of the minimum value is not possible)

**Comment by MariusJulius** [comment]
@KaarelKa as I remember BE doesn't accept minimum price at the moment, so fixed needed on BE side?

**Comment by KaarelKa** [comment]
@MariusJulius Yeah, I will disable this field in FE

**Comment by KaarelKa** [comment]
@MariusJulius @plakitkelly Okay this was actually already removed before the last deploy. So there is nothing more we can do on FE right now. We can add it back, once BE allows the saving of minimum value.

**Comment by MariusJulius** [comment]
@PlaksoBirgit is this minimum fee as a separate unit needed at all? In case price is lower than minimum fee, it uses the vendor minimum price. No point to set this manually as it is only relevant to vendor - not to get below the fee. Maybe add extra information about minimal fee - which will be not editable (or only if minimal fee is higher than end price).

**Comment by PlaksoBirgit** [comment]
@MariusJulius has there been extra logic added that we can't add below minimum price (vendor always has to get at least the miminum fee)? 
the initial idea was that PM can use it if the work load is small (they can also then easily see what is the minimum for this specific vendor)

**Comment by MariusJulius** [comment]
@PlaksoBirgit so to confirm as this behaves a bit different then other pricing options with price x unit. Hinna väärtus (EUR) / unit price is the field where minimum fee is entered. Quantity field is locked for editing and total shows the same as unit price (the min fee).

**Comment by PlaksoBirgit** [comment]
@MariusJulius yes, Hinna väärtus is the place where you can enter the minimum fee and we can disable the quantity fee or if it is easier to make for FE then just add quantity "1" and leave it editable

**Comment by MariusJulius** [comment]
@kadmit please check if comments are ok and we can proceed?

**Comment by MariusJulius** [comment]
Min price is not included in the calculations as other prices (eg summing up pages, words etc) - it will be done in the later phases. Min fee also doesn't replace any other price if there is no value.

If there is only one unit (minimal fee) added then cost is calculated based on minimal fee. E.g PM has set minimal fee 20 and quantity 2 and if Vendor has minimal fee value X in price list then this is multiplied 2X and shown in the cost par next to vendor. Vendor minimal fee doesn't overwrite the PM added minimal fee in the volume modal.

**Comment by MariusJulius** [comment]
I suppose we need to enable it on FE side also.
